src/components/ResultsBody/PaginatedList/index.jsx

```diff
@@ -28,6 +31,12 @@ export const PaginatedList = ({ limit }) => {
     );
   }
 
+
+  if (contextIsError) {
+    return <></>;
```

Comment:
I already pointed out not to return fragment. You can return null instead

Reply:
done